cypher/cypher-docs/src/docs/dev/ql/administration/security/list-users-table-columns.asciidoc

```diff
@@ -1,34 +0,0 @@
-.List users output
```

Comment:
neo4j-documentation/4.3/cypher/cypher-docs/target/docs/dev/ql/administration/security/administration-security-users-and-roles.adoc